### ISSUE TITLE
This calculation of plus operation is right?

### ISSUE THREAD

**Issue by Eomtaesan** [Microsoft/CNTK#2693]
There are two matrix: A = [256 : 256 : 1000], B = [1 : 1 : 1000]

I want to make this result:
For i = 1 to 1000
{
A[all : all : i] += B[1 : 1 : i]
}

In brainscript, I write this operation: result = A + B
Then the result has intended output?

**Comment by ebarsoumMS** [comment]
Yes, this will broadcast the B to A. Are you running into perf problem for the transpose?
